src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,110 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final String INSERT = "INSERT INTO manufacturers (name, country) VALUES (?, ?)";
+    private static final String GET_BY_ID =
+            "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = false";
```

Comment:
```suggestion
            "SELECT * FROM manufacturers WHERE id = ? AND is_deleted = FALSE";
```
Let's use sql keywords in uppercase format. Please fix all cases.